public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -485,7 +486,7 @@ export class ElasticDatasource extends DataSourceApi<ElasticsearchQuery, Elastic
    * search_after feature.
    */
   showContextToggle(): boolean {
-    return this.esVersion > 5;
+    return gte(this.esVersion, '5.0.0');
```

Comment:
It was `>` before, now it's `>=`. Is it intended?

Reply:
yeah, it slipped through at some point, `search_after` is actually supported by `5.0.0` : https://www.elastic.co/guide/en/elasticsearch/reference/5.0/search-request-search-after.html

Reply:
![nooice](https://user-images.githubusercontent.com/745532/117155988-8b27d600-adbd-11eb-8560-a6f17d5d0c94.gif)